Data/DataModel/Response.cs

```diff
@@ -1,18 +1,53 @@
-﻿using System;
-using System.Collections.Generic;
-using System.ComponentModel.DataAnnotations.Schema;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using System.ComponentModel.DataAnnotations.Schema;
+using System.Text.RegularExpressions;
 
 namespace ServerCore.DataModel
 {
+    /// <summary>
+    /// The response to a particular submission for a given puzzle
+    /// </summary>
     public class Response
     {
+        private string submittedText = string.Empty;
+
+        /// <summary>
+        /// The Id
+        /// </summary>
         [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int ID { get; set; }
+
+        /// <summary>
+        /// The puzzle the response is for
+        /// </summary>
         public virtual Puzzle Puzzle {get; set;}
+
+        /// <summary>
+        /// True if the submitted text is considered a solution to the puzzle
+        /// </summary>
         public bool IsSolution { get; set; }
-        public string SubmittedText { get; set; }
+
+        /// <summary>
+        /// The submitted text
+        /// </summary>
+        public string SubmittedText
+        {
+            get { return this.submittedText; }
```

Comment:
This pattern doesn't cause EF to freak out? [Note: I have no EF experience, just trying to learn what the boundaries are]

Reply:
Not that I know of.

Reply:
@asyasky says the standard for fields should be camel case with no 'this.' In this case, that would be: ```get { return submittedText; }```